Io/KeyboardActionMapping.cpp

```diff
@@ -88,6 +88,7 @@ GameKey KeyboardActionMapping::MapDefaultKey(InputAction action) {
             return keyMappingParams[i].m_key;
         }
     }
+    __debugbreak();
```

Comment:
Why that here?

Reply:
If we got here then we're not returning anything from a non-void function, and this is a problem.

I'm not sure what would be better to use here. Maybe Assert(false)?

Reply:
return GameKey::None?

Reply:
Good idea, thanks. Pushed a fix.